src/main/java/org/asamk/signal/Main.java

```diff
@@ -622,6 +638,17 @@ private static void printIdentityFingerprint(Manager m, String theirUsername, Js
         System.out.println(String.format("%s: %s Added: %s Fingerprint: %s Safety Number: %s", theirUsername,
                 theirId.trustLevel, theirId.added, Hex.toStringCondensed(theirId.getFingerprint()), digits));
     }
+    
+    private static void printGroup(GroupInfo group, boolean detailed) {
+        System.out.println(String.format("Group id: %s\n Group name: %s \n active: %s",
```

Comment:
I'd prefer having each group in a single line, e.g.:
"Id: %s Name: \"%s\" Active: %s Members: ..."